huggingface/pytorch/tgi/docker/2.0.1/Dockerfile

```diff
@@ -0,0 +1,261 @@
+# Rust builder
```

Comment:
@philschmid it also requires changes to License and releases.json update. Reference PR: https://github.com/awslabs/llm-hosting-container/pull/69 . We have merged TGI 2.0.1 pr, so you can cancel this PR, but sharing the reference for future PRs.